subgraphs/_reference_/src/sdk/protocols/bridge/account.ts

```diff
@@ -190,7 +190,7 @@ export class Account {
   transferIn(
     pool: Pool,
     route: PoolRoute,
-    source: Address,
+    source: Bytes,
```

Comment:
Same comment about changing the API as above. Make a new method instead, for transfer in and out for non evm chains.

Reply:
same as above.

Reply:
Resolving to keep the conversation on the other comment